jsapp/js/components/common/multiCheckbox.tsx

```diff
@@ -1,23 +1,33 @@
 import React from 'react';
-import bem from 'js/bem';
+import bem, {makeBem} from 'js/bem';
 import Checkbox from 'js/components/common/checkbox';
 import './multiCheckbox.scss';
 
-interface MultiCheckboxItem {
+bem.MultiCheckbox = makeBem(null, 'multi-checkbox', 'ul');
+bem.MultiCheckbox__item = makeBem(bem.MultiCheckbox, 'item', 'li');
+// This wrapper adds the visible frame around all checkboxes.
+bem.MultiCheckbox__wrapper = makeBem(bem.MultiCheckbox, 'wrapper');
+
+export interface MultiCheckboxItem {
   /** any other properties will be passed back with onChange */
   [propName: string]: any;
   checked: boolean;
-  disabled: boolean;
+  disabled?: boolean;
   label: string;
 }
 
 interface MultiCheckboxProps {
   items: MultiCheckboxItem[];
+  /** Use this to disable all checkboxes - useful for blocking changes while loading. */
+  disabled?: boolean;
   /** Returns whole list whenever any item changes */
   onChange: (items: MultiCheckboxItem[]) => void;
 }
 
-/** A MultiCheckbox generic component. */
+/**
+ * A MultiCheckbox generic component.
+ * Use optional `bem.MultiCheckbox__wrapper` to display a frame around it.
```

Comment:
I wonder why not put the wrapper around the component, and have an optional prop enable the wrapper in the component itself?

Reply:
Good idea, changed it by adding a `type` prop :)